bokeh/core/_templates/zeppelin.html

```diff
@@ -0,0 +1,12 @@
+{#
+Renders for zeppelin notebook.
+#}
+
+<div>
+    {{ plot_div|indent(8) }}
+    {{ plot_script|indent(8) }}
+</div>
+<div>
+    {{ bokeh_css }}
+    {{ bokeh_js }}
+</div>
```

Comment:
This will probably "work". I think it's worth considering if the existing "autoload" style machinery can be used or lightly adapted. I don't know a lot about the Zeppelin notebook , but I am guessing you might run into the same race condition that happened in the jupyter notebook: namely, if you "run all" cells then a later cell that relies on BokehJS being loaded might exectute before the browser actually finishes loading BokehJS.